test/libsolidity/ASTJSON/override.json

```diff
@@ -0,0 +1,337 @@
+	[
+		{
+			"baseContracts" : [],
+			"contractDependencies" : [],
+			"contractKind" : "contract",
+			"documentation" : null,
+			"fullyImplemented" : true,
+			"id" : 5,
+			"linearizedBaseContracts" : 
+			[
+				5
+			],
+			"name" : "A",
+			"nodeType" : "ContractDefinition",
+			"nodes" : 
+			[
+				{
+					"body" : 
+					{
+						"id" : 3,
+						"nodeType" : "Block",
+						"src" : "36:2:1",
+						"statements" : []
+					},
+					"documentation" : null,
+					"id" : 4,
+					"implemented" : true,
+					"kind" : "function",
+					"modifiers" : [],
+					"name" : "faa",
+					"nodeType" : "FunctionDefinition",
+					"overrides" : null,
+					"parameters" : 
+					{
+						"id" : 1,
+						"nodeType" : "ParameterList",
+						"parameters" : [],
+						"src" : "26:2:1"
+					},
+					"returnParameters" : 
+					{
+						"id" : 2,
+						"nodeType" : "ParameterList",
+						"parameters" : [],
+						"src" : "36:0:1"
+					},
+					"scope" : 5,
+					"src" : "14:24:1",
+					"stateMutability" : "nonpayable",
+					"superFunction" : null,
+					"visibility" : "public"
+				}
+			],
+			"scope" : 32,
+			"src" : "0:40:1"
+		},
+		{
+			"baseContracts" : 
+			[
+				{
+					"arguments" : null,
+					"baseName" : 
+					{
+						"contractScope" : null,
+						"id" : 6,
+						"name" : "A",
+						"nodeType" : "UserDefinedTypeName",
+						"referencedDeclaration" : 5,
+						"src" : "55:1:1",
+						"typeDescriptions" : 
+						{
+							"typeIdentifier" : "t_contract$_A_$5",
+							"typeString" : "contract A"
+						}
+					},
+					"id" : 7,
+					"nodeType" : "InheritanceSpecifier",
+					"src" : "55:1:1"
+				}
+			],
+			"contractDependencies" : 
+			[
+				5
+			],
+			"contractKind" : "contract",
+			"documentation" : null,
+			"fullyImplemented" : false,
+			"id" : 16,
+			"linearizedBaseContracts" : 
+			[
+				16,
+				5
+			],
+			"name" : "B",
+			"nodeType" : "ContractDefinition",
+			"nodes" : 
+			[
+				{
+					"body" : null,
+					"documentation" : null,
+					"id" : 10,
+					"implemented" : false,
+					"kind" : "function",
+					"modifiers" : [],
+					"name" : "foo",
+					"nodeType" : "FunctionDefinition",
+					"overrides" : null,
+					"parameters" : 
+					{
+						"id" : 8,
+						"nodeType" : "ParameterList",
+						"parameters" : [],
+						"src" : "72:2:1"
+					},
+					"returnParameters" : 
+					{
+						"id" : 9,
+						"nodeType" : "ParameterList",
+						"parameters" : [],
+						"src" : "81:0:1"
+					},
+					"scope" : 16,
+					"src" : "60:22:1",
+					"stateMutability" : "nonpayable",
+					"superFunction" : null,
+					"visibility" : "public"
+				},
+				{
+					"body" : 
+					{
+						"id" : 14,
+						"nodeType" : "Block",
+						"src" : "115:2:1",
+						"statements" : []
+					},
+					"documentation" : null,
+					"id" : 15,
+					"implemented" : true,
+					"kind" : "function",
+					"modifiers" : [],
+					"name" : "faa",
+					"nodeType" : "FunctionDefinition",
+					"overrides" : [],
```

Comment:
What's the difference between `"overrides" : []` and `"overrides" : null` ?

Reply:
I believe `[]` means that `override` was specified but no list of things overriden was. E.g.

```
function foo() public overrides {}  // overrides: []
function foo() public overrides(A, B) {} // overrides: ["A", "B"]
function foo() public {} // overrides: null
```

Reply:
Makes sense, thanks!